crates/pallet-bridges-initializer-currency-swap/src/mock/mod.rs

```diff
@@ -0,0 +1,48 @@
+//! The mocks for the pallet testing.
+
+use frame_support::{parameter_types, sp_io, sp_runtime::BuildStorage};
+
+use crate::{self as pallet_bridges_initializer_currency_swap};
+
+pub mod v1;
+pub mod v2;
+
+pub(crate) const EXISTENTIAL_DEPOSIT_NATIVE: u64 = 10;
+pub(crate) const EXISTENTIAL_DEPOSIT_EVM: u64 = 20;
+
+pub(crate) type AccountId = u64;
+pub(crate) type EvmAccountId = u64;
+
+type Balance = u64;
+
+type BalancesInstanceNative = pallet_balances::Instance1;
+type BalancesInstanceEvm = pallet_balances::Instance2;
+
+parameter_types! {
+    pub NativeTreasury: AccountId = 4200;
+}
```

Comment:
Use ConstU64 instead?

Reply:
I prefer use `parameter_types` to indicate that it's `AccountId`, imo.

Reply:
My idea was to avoid sharing this bit, so we can inline it, but this works too